src/main/java/com/philips/research/spdxbuilder/persistence/ort/OrtReader.java

```diff
@@ -30,137 +30,137 @@
-            final var tick = projectIds.contains(project.id) ? "+" : "-";
-            final var from = (project.definitionFilePath != null) ? " from '" + project.definitionFilePath + "'" : "";
-            System.out.println(tick + " '" + project.id + "'" + from);
-        });
-    }
-
-    private void cleanupYaml(OrtJson yaml, Map<String, URI> projectPackages, Map<String, List<String>> projectExcludes) {
-        if (yaml.repository == null) {
-            return;
-        }
-        //noinspection ConstantConditions
-        final var result = yaml.analyzer.result;
-        final var excludedPaths = yaml.repository.getExcludePaths();
-        final var excludedScopes = yaml.repository.getExcludeScopes();
-
-        assert result != null;
-        result.removeProjects(excludedPaths);
-        result.keepProjects(projectPackages.keySet());
-        result.updateProjectPackages(projectPackages);
-        result.projects.forEach(p -> {
-            final var globs = new HashSet<>(excludedScopes);
-            projectExcludes.getOrDefault(p.id, List.of()).stream()
-                    .map(pattern -> FILE_SYSTEM.getPathMatcher("glob:" + pattern))
-                    .forEach(globs::add);
-            p.removeScopes(globs);
-        });
-    }
-
-    private void registerProjects(ResultJson result, BillOfMaterials bom, HashMap<String, Package> dictionary) {
-        result.projects.forEach(p -> {
-            if (p.id == null) {
-                return;
-            }
-            System.out.println();
-            System.out.println("Adding project '" + p.id + "':");
-            var project = p.createPackage();
-            dictionary.put(p.id, project);
-            bom.addPackage(project);
-            p.scopes.forEach(scope -> {
-                System.out.println("+ Adding scope '" + scope.name + "'");
-                scope.putAllDependencies(dictionary);
-            });
-        });
-    }
-
-    private void registerPackages(ResultJson result, BillOfMaterials bom, HashMap<String, Package> dictionary) {
-        result.packages()
-                .filter(pkg -> dictionary.containsKey(pkg.id))
-                .forEach(pkg -> {
-                    if (pkg.id == null) {
-                        return;
-                    }
-                    final var created = pkg.createPackage();
-                    dictionary.put(pkg.id, created);
-                    bom.addPackage(created);
-                });
-    }
-
-    private void registerRelations(ResultJson result, BillOfMaterials bom, HashMap<String, Package> dictionary) {
-        result.projects.forEach(p -> {
-            final var project = dictionary.get(p.id);
-            p.scopes.stream()
-                    .flatMap(scope -> scope.dependencies.stream())
-                    .forEach(dep -> dep.registerRelations(bom, project, dictionary));
-        });
-    }
+  private static final FileSystem FILE_SYSTEM = FileSystems.getDefault();
```

Comment:
Let's not reformat to a different indenting; all code currently uses a standard indent of 4 spaces.